db/migrate/20240209113122_remove_globus_uuid_index.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+class RemoveGlobusUuidIndex < ActiveRecord::Migration[6.1]
+  def change
+    remove_index :allocator, column: :globus_uuid
```

Comment:
I guess we might need this index, because we are using this column in serializer at many places, you can take a look at this commit for reference https://github.com/datacite/lupo/commit/eeb811c70c10b33c634d61f9e9ed23e9b352993c

Reply:
@ashwinisukale unfortunately we are not using the column to fetch any data(that is either `.where`, `find_by` or `order_by`), inside the serialiser it is used to conditionally include the the attribute.